highfidelity/hifi
# FB11109 Preview disabled image swap (Commerce only)

*** Test plan *** 

1. Ensure wallet-specific 'disabled preview' image is shown when executing security-related actions in wallet (entering wallet password, setting up security image) 

![preview-privacy](https://user-images.githubusercontent.com/23638/35064627-c778e35c-fbdb-11e7-8b21-b5b3786dec76.png)

2. Ensure wallet operations doesn't change preview image if preview was already disabled (using menu or due to v-sync enabled) before entering wallet

Additional test case: 

1. Enter wallet with enabled preview
2. Ensure preview turns off every time wallet prompts to type sensitive information (password, security image) and turns on after finishing with sensitive information.

Bonus: 

1. Enter wallet with enabled preview. 
2. Ensure wallet-specific 'disabled preview' appeared
3. Re-enable preview 
4. Leave wallet by clicking 'cancel' button
5. Disable preview
6. Ensure normal version of 'disabled preview' (not wallet-specific) is shown

Bonus2: 

1. Enter wallet with enabled preview.
2. Ensure wallet-specific 'disabled preview' appeared
3. Enable preview
4. Disable preview
5. Ensure wallet-specific 'disabled preview' appeared
